src/main/java/task/Task.java

```diff
@@ -0,0 +1,51 @@
+package task;
+
+public class Task {
```

Comment:
leave out public getters and setter if they are not used for better security and assess